packages/contracts/tasks/deposits.ts

```diff
@@ -0,0 +1,54 @@
+import { task, types } from 'hardhat/config'
+import { Contract, providers, utils, Wallet } from 'ethers'
+import dotenv from 'dotenv'
+
+import * as depositFeedArtifact from '../artifacts/contracts/L1/DepositFeed.sol/DepositFeed.json'
+
+task('deposit', 'Deposits funds unto Optimism bedrock.')
```

Comment:
`unto` -> `into`

Reply:
also do we want to keep the name bedrock forever? i thought it was the release, then new releases would have new names